src/metrics/kbet_pg/script.py

```diff
@@ -0,0 +1,64 @@
+from tqdm import tqdm
+import sys
+import numpy as np
+import anndata as ad
+import pegasus as pg
+import pegasusio
+from scipy.sparse import csr_matrix
+
+
+def compute_kbet(mmdata, *args, **kwargs):
+    stat_mean, pvalue_mean, accept_rate = pg.calc_kBET(mmdata, *args, **kwargs)
+    return accept_rate
+
+
+## VIASH START
+par = {
+    'input_integrated': 'resources_test/task_batch_integration/cxg_immune_cell_atlas/integrated_full.h5ad',
+    'output': 'output.h5ad',
+}
+
+meta = {
+    'name': 'kbet_pg',
+}
+## VIASH END
+
+sys.path.append(meta["resources_dir"])
+from read_anndata_partial import read_anndata
+
+n_threads = meta.get("cpus", -1)
```

Comment:
Right! This should fix it

```suggestion
n_threads = meta["cpus"] or -1